Makefile

```diff
@@ -16,3 +16,8 @@ init:
 
 	# ensure there's a .env file present
 	stat .env 2>/dev/null || echo "SENTRY_ORG=<your org slug>\nSENTRY_PROJECT=<your project slug>" > .env
+
+release:
+	CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO xcodebuild -workspace EmpowerPlant.xcworkspace -scheme EmpowerPlant -configuration Release -derivedDataPath build -sdk iphoneos clean build 2>&1 | tee release-build.log | xcbeautify
+	zip -r EmpowerPlant_release.zip ./build/Build/Products/Release-iphoneos/EmpowerPlant.app
+	@echo "\nBuild completed. Create a new release in GitHub and upload ./EmpowerPlant_release.zip."
```

Comment:
Here's the `make release` task. Wanted to call it out since most of the diff is lockfile changes and it's kinda buried.